Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM ubuntu:latest
+MAINTAINER Gergely Daroczi <gergely@system1.com>
+
+ENV GRPC_RELEASE_TAG v1.4.5
+ENV PROTOC_RELEASE_TAG v3.2.0
+ENV R_BASE_VERSION 3.4.2
+
+## install build tools
+RUN apt-get update && apt-get install -y \
+  build-essential autoconf libtool \
+  libgflags-dev libgtest-dev clang libc++-dev \
+  unzip git curl wget \
+  pkg-config locales && \
+  apt-get clean && rm -rf /var/lib/apt/lists/
+
+## build grpc and protobuf
+RUN git clone -b ${GRPC_RELEASE_TAG} https://github.com/grpc/grpc /var/local/git/grpc && \
+  cd /var/local/git/grpc && git submodule update --init && \
+  cd /var/local/git/grpc/third_party/protobuf && \
+  git checkout ${PROTOC_RELEASE_TAG} && \
+  ./autogen.sh && ./configure && \
+  make && make install && make clean && ldconfig && \
+  cd /var/local/git/grpc && \
+  make && make install && make clean && ldconfig
```

Comment:
Would it be possible to get this working with the debian/ubuntu prebuilt binaries? https://packages.debian.org/search?keywords=libgrpc-dev
https://packages.ubuntu.com/search?keywords=libgrpc-dev

Instead of building all dependencies from scratch.